sql/driver/driver_test.go

```diff
@@ -202,6 +204,50 @@ func TestPlaceholders(t *testing.T) {
 	}
 }
 
+func TestConnectionSettings(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	s := server.StartTestServer(nil)
+	url := "https://root@" + s.ServingAddr() + "?certs=test_certs"
+	// Create a new Database called t.
+	db, err := sql.Open("cockroach", url)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if _, err := db.Exec(`CREATE DATABASE t`); err != nil {
+		t.Fatal(err)
+	}
+	_ = db.Close()
```

Comment:
Perhaps we should add a `create-if-not-exists` parameter to the URL to ease database creation for testing purposes.